### ISSUE TITLE
Page-Up, Page-Down keys not working in full-screen view.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#7]
```
What steps will reproduce the problem?
1. Click FlexPaper's internal full-screen icon to go to full-screen view.
2. Press the page-up or page-down key.

What is the expected output? What do you see instead?

I expected the page to scroll up or down by some incremental amount. 
Nothing happened.

What version of the product are you using? On what operating system?

FlexPaper 1.2 on Fedora 11.

Please provide any additional information below.

I've only tried this on different browsers (Chrome and Firefox) and with
different versions of the libflashplayer.so so I'm guessing this is a bug
in the FlexPaper code.


```

Original issue reported on code.google.com by `gregory....@gmail.com` on 3 Feb 2010 at 6:59

**Comment by GoogleCodeExporter** [comment]
```
Yeah I've noticed this - most keys are actually disabled in full screen as a 
security 
measure by the flash player and there isn't much I can do about that. Like I 
said in 
another thread, might be better to open a new maximized browser window with 
flexpaper 
in it rather than the current solution. Kind of annoying though. I think I'll 
add that 
as an option/parameter to the viewer, like so:

FullscreenAsMaximizedWin (true/false)
```

Original comment by `erik.eng...@gmail.com` on 3 Feb 2010 at 7:12

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@gmail.com` on 21 Feb 2010 at 10:44
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed in 1.2.2 by use of the new "FullScreenAsMaxWindow" property
```

Original comment by `erik.eng...@gmail.com` on 16 Mar 2010 at 6:41
- Changed state: **Fixed**
